activemodel/test/cases/attribute_set_test.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 require "cases/helper"
+require "active_model/attribute_set"
```

Comment:
If we need this require maybe we are missing some autoload. are those files being required in `lib`?